src/features/fwiCalculator/components/TimeRangeSlider.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import IOSSlider from 'features/fwiCalculator/components/IOSSlider'
+import { InputLabel, makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles({
+  root: {
+    marginTop: 20
+  }
+})
+
+interface Props {
+  timeRange: number
+  onYearRangeChange: (yearRangeNumber: number) => void
+}
+
```

Comment:
Maybe create two variables called min and max here, then use them in TIME_RANGE_OPTIONS and IOSSlider?